.circleci/config.yml

```diff
@@ -364,6 +389,12 @@ workflows:
       - upload_coverage:
           requires:
             - test_cover
+      - reproducible_builds:
+          filters:
+            branches:
+              only:
+                - master
+                - develop
```

Comment:
we don't have develop anymore

Reply:
removed thanks

There is another occurrence here:

```
363       - deploy_docs:
364           filters:
365             branches:
366               only:
367                 - master
368                 - develop
```

Should I remove that too?

Reply:
Yes please

Reply:
done